app/Imports/WordsImport.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Imports;
+
+use App\Models\User;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Auth;
+use Maatwebsite\Excel\Concerns\WithStartRow;
+use Maatwebsite\Excel\Concerns\ToCollection;
+use Illuminate\Support\Facades\DB;
+use Exception;
+
+class WordsImport implements ToCollection, WithStartRow
+{
+    public function collection(Collection $rows)
+    {    
+        $rows = $rows->toArray();
+        sort($rows);
+        foreach ($rows as $key => $row) {
+            if (($key >= 1) && ($row[0] == $rows[$key-1][0]) && ($row[1] == $rows[$key-1][1])) {
+                unset($rows[$key-1]);
+            }
+        }
+        $userId = Auth::id();
+        $user = User::findOrFail($userId);
```

Comment:
sao lại find như này em?
```suggestion
        $user = Auth::user();
```

Reply:
nếu dùng $user = Auth::user() thì biến $user không thể chứa các relationship được ạ

Reply:
sao lại ko lấy được realation ship nhỉ? vẫn ->realate được mà